src/components/Card.js

```diff
@@ -73,56 +80,71 @@ export const Card = () => {
         >
           <QRCode
             className="qrcode"
-            style={{ paddingRight: portrait ? '' : '1em' }}
+            style={{
+              paddingRight: portrait ? '' : '1em',
+              height: currentSize + 'px',
```

Comment:
Prefer template literals to string concatenation:
```js
height: `${size.size}px`,
```

Reply:
Done. Thanks!